### PR TITLE
fix up tests

### PR DESCRIPTION
**Pull request by rhatdan**

### CODE REVIEW

**Comment by rhatdan** [comment]
@ericcurtin This adds fixes, but I still don't believe tests are running?

**Comment by ericcurtin** [comment]
I think it's because you are using tabs instead of spaces @rhatdan , but that's a guess

**Comment by ericcurtin** [comment]
If we convert all tabs to spaces in:

.github/workflows/ci.yml

I think it will run...

File: ramalama/cli.py [hunk 1]
```diff
@@ -203,9 +204,12 @@ def _list_containers(args):
     if conman == "":
         raise IndexError("no container manager (Podman, Docker) found")
 
-    conman_args = [conman, "ps", "-a", "--filter", "label=RAMALAMA container"]
+#    conman_args = [conman, "ps", "-a", "--filter", "label=RAMALAMA container"]
+    conman_args = [conman, "ps"]
```
[the remark this answers is not in the document]
Comment (rhatdan) [reply ramalama/cli.py:208]: Nope that was a debugging thing.  Good catch.

**Review by ericcurtin — APPROVED** [review]

**Pull request merged**